src/pages/Main/BestSeller/BestSeller.js

```diff
@@ -0,0 +1,105 @@
+import { useState } from 'react';
+import './BestSeller.scss';
+import BestSellerCard from './BestSellerCard/BestSellerCard';
+
+function BestSeller() {
+  const [currentPosition, setCurrentPosition] = useState(0);
+  const [transitionTime, setTransitionTime] = useState(0.3);
+  const cardSize = 23;
+
+  const handleClickLeftButton = () => {
+    if (currentPosition === 0) {
+      setTransitionTime(0);
+      setCurrentPosition(CardsList.length - 1 + 1);
+      setTimeout(() => {
+        setTransitionTime(0.3);
+        setCurrentPosition(CardsList.length - 1);
+      }, 10);
+    } else {
+      setCurrentPosition(prev => prev - 1);
+    }
+  };
+  const handleClickRightButton = () => {
+    if (currentPosition === CardsList.length - 1) {
+      setTransitionTime(0);
+      setCurrentPosition(-1);
+      setTimeout(() => {
+        setTransitionTime(0.3);
+        setCurrentPosition(0);
+      }, 0);
+    } else {
+      setCurrentPosition(prev => prev + 1);
+    }
+  };
+
+  return (
+    <div className="bestSeller">
+      <div className="header">
+        <h1 className="title">Best Seller</h1>
+        <button className="linkButton">
+          더 많은 제품 보기&nbsp;&nbsp;&nbsp;>
+        </button>
+      </div>
+      <div
+        className="slide"
+        style={{
+          transform: `translateX(${(currentPosition - 1) * -cardSize}rem)`,
+          transition: `all ${transitionTime}s`,
+        }}
+      >
+        {dummyCardsList.map((card, idx) => {
+          const hasTwoCard = card.imgUrl.length === 2;
+          return (
+            <BestSellerCard
+              key={idx}
+              hasTwoCard={hasTwoCard}
+              imgUrl={card.imgUrl}
+            />
+          );
+        })}
+      </div>
+      <div className="slideButtonContainer">
+        <button onClick={handleClickLeftButton}>
+          <i className="fa-solid fa-angle-left fa-xl" />
+        </button>
+        <ul className="slideStateContainer">
+          {CardsList.map((card, idx) => {

```

Comment:
매개변수로 card를 선언해주셨는데 사용되지 않고 있네요.
이럴 때는 굳이 매개변수로 작성하지 않아도 _ 로 해주셔도 됩니다.
개발자들이 주로 사용하기도 하는데 첫번째 인자가 크게 중요하지 않다고 생각할 수 있습니다.
```suggestion
          {CardsList.map((_, idx) => {

```

Reply:
해당 리뷰 캡쳐해서 유진님 아이디 가린 뒤 블로그에 올려도 되나요??!!! 👍👍

Reply:
넵넵ㅋㅋㅋㅋㅋ